pkg/event/config.go

```diff
@@ -0,0 +1,70 @@
+package event
+
+type KafkaConfig struct {
+	Timeout int
+	Group   struct {
+		Id string
+	}
+	Auto struct {
+		Offset struct {
+			Reset string
+		}
+		Commit struct {
+			Interval struct {
+				Ms int
+			}
+		}
+	}
+	Bootstrap struct {
+		Servers string
+	}
+	Topics []string
+	Sasl   struct {
+		Username  string
+		Password  string
+		Mechanism string
+		Protocol  string
+	}
+	Request struct {
+		Timeout struct {
+			Ms int
+		}
+		Required struct {
+			Acks int
+		}
+	}
+	Capath  string
+	Message struct {
+		Send struct {
+			Max struct {
+				Retries int
+			}
+		}
+	}
+	Retry struct {
+		Backoff struct {
+			Ms int
+		}
+	}
+}
```

Comment:
since this is more config related, shouldn't it be in ./config?

Reply:
i should say, i think all the config structs should probably live in ./config/

Reply:
Thanks and sorry for the delay! the reason I moved this part of the struct was a circular dependency; I don't think the solution I have provided is the right one to remove that circular dependency, but I have to learn more about how to break the cycles of the dependency graph. I am open to suggestions on this; I recognize this will need refactor.